src/commands/project/create.ts

```diff
@@ -7,12 +7,13 @@ import { getContract, getSigner, parseAddress, parseHash, pretty, run } from "..
 export default class ProjectCreate extends TransactionCommand {
   static summary = "Register a new project on the Armada Network.";
   static examples = ['<%= config.bin %> <%= command.id %> "My Project" notify@myproject.com'];
-  static usage = "<%= command.id %> [--owner ADDR] NAME EMAIL [URL] [SHA]";
+  static usage = "<%= command.id %> [--owner ADDR] NAME EMAIL [URL] [SHA] [METADATA]";
   static args: Arg[] = [
     { name: "NAME", description: "The human readable name of the new project.", required: true },
     { name: "EMAIL", description: "The project email for admin notifications.", required: true },
     { name: "URL", description: "The public URL to fetch the content bundle.", default: "" },
     { name: "SHA", description: "The SHA-256 checksum of the content bundle.", default: "" },
+    { name: "METADATA", description: "Optional JSON.stringified metadata to attach to this project.", default: "" },
```

Comment:
Also curious, is there a good reason why this should not automatically stingify it for the user? Would then also require updating "project show" and "project list" commands to automatically unstringify it.

Reply:
@the-masthead the reason i didn't automatically stringify is because command line args are input and parsed as strings and it seems more straightforward to keep the API as passthrough. the alternative is to have to validate the JSON string and then pass it to the metadata contract function. do you prefer this approach?

Reply:
> Also curious, is there a good reason why this should not automatically stingify it for the user? Would then also require updating "project show" and "project list" commands to automatically unstringify it.

@the-masthead the reason i didn't automatically stringify is because command line args are input and parsed as strings and it seems more straightforward to keep the API as passthrough. the alternative is to have to validate the JSON string and then pass it to the metadata contract function. do you prefer this approach?